src/.env

```diff
@@ -0,0 +1 @@
+API_SERVER_URL = "http://localhost:3001";
```

Comment:
@Alihaider168 can we get rid of this file if this variable is being set in the config file ?